### PR TITLE
Filter button styles

### PR DESCRIPTION
**Pull request by ricardoom**

### CODE REVIEW

File: docs/_sections/buttons.html [hunk 1]
```diff
@@ -76,7 +76,92 @@ <h3 id="linkButton">Link Button</h3>
...
+  &lt;/svg&gt;
+&lt;/button&gt;
+  </code></pre>
+</div>
+<!-- ////// -->
+<h3 id="cancelFilters">Cancel Filter Button</h3>
+  <p>In a scenario where we have filters activated, we need to allow the user to unset each filter. The "x" icon is added inline and uses the <code>nypl-icon</code> class. </p>
+  <div class="nypl-toolkit-example">
+    <span class="nypl-example-text">Example</span>
+      <button class="nypl-unset-filter">Filter
```
Comment (EdwinGuzman) [inline docs/_sections/buttons.html]: Maybe this should say 'cancel' to go along with the example.

Comment (ricardoom) [reply docs/_sections/buttons.html]: `nypl-cancel-filter`?

Comment (EdwinGuzman) [reply docs/_sections/buttons.html]: Oh nevermind. I thought that "Filter" was the label but it's a placeholder for the name of any filter. All's good. 👍

Comment (ricardoom) [reply docs/_sections/buttons.html]: This is the design pattern...
screen shot 2017-09-13 at 4 17 32 pm

Comment (ricardoom) [reply docs/_sections/buttons.html]: regarding the checkbox style. I have a fix in on `checkbox-style` branch...

Comment (EdwinGuzman) [reply docs/_sections/buttons.html]: I'd say fix the conflicts for this PR and merge it and then update.

**Pull request merged**